gc/verbose/schema.xsd

```diff
@@ -34,6 +34,7 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 	<element name="arraylet-unknown" type="vgc:arraylet-unknown" />
 	<element name="cpu-util" type="vgc:cpu-util" />
 	<element name="numa" type="vgc:numa" />
+	<element name="offheap" type="vgc:offheap" />
```

Comment:
I see inconstancy, sometimes  it's offheap-objects and sometimes jiust offheap
probably offheap-objects is better

Reply:
ah, seems like both are needed... one is part of mem-info and the other one is for gc-op

Reply:
but I'd reverse the names

from what I see, we never use '...-objects' in gc-ops
and we do used '...-objects' in mem-info